Dockerfile

```
@@ -243,9 +243,10 @@ RUN apk --update --no-cache add \
     tzdata \
     unzip \
     util-linux \
-    zip \
-  && pip3 install --upgrade pip \
-  && pip3 install cfscrape cloudscraper \
+    zip
+
+RUN pip3 install --upgrade --break-system-packages pip \
```

Can you keep a single `RUN`? I want to avoid extra cached layers otherwise would exceed 10GB cache size with GitHub Actions Cache backend.

```suggestion
    zip \
  && pip3 install --upgrade --break-system-packages pip \
```